xsuite/src/world/fsworld.test.ts

```diff
@@ -223,6 +223,30 @@ test.concurrent("FSWorld.getAccountBalance", async () => {
   expect(await world.getAccountBalance(wallet)).toEqual(1234n);
 });
 
+test.concurrent("FSWorld.getAccountValue - Invalid key", async () => {
+  using world = await FSWorld.start();
+  const wallet = await world.createWallet({ kvs: [[e.Str("n"), e.U(1)]] });
+  await expect(world.getAccountValue(wallet, "n")).rejects.toThrow(
+    "invalid byte",
+  );
+});
+
+test.concurrent("FSWorld.getAccountValue - Non-existent key", async () => {
```

Comment:
I'm not sure that it's worth testing this case, where the account doesn't have any kvs?

Let's rather test here the case where the account has some kvs and the key we request is not present in the kvs. I guess it should return an empty string?

Reply:
Also let's call the test `FSWorld.getAccountValue - non-present key`